perun-rpc/src/main/perl/Perun/beans/User.pm

```diff
@@ -181,8 +181,8 @@ sub setIsServiceUser
 	my $self = shift;
 	my $value = shift;
 	$value eq 'true' || $value eq 1 || ref $value eq "JSON::PP::Boolean"
```

Comment:
Please rebase this branch on current master. Zora updated listOfPrincipalRoles tool, which was not working properly.